src/Tizen.NUI.Components/Controls/Progress.cs

```diff
@@ -152,6 +152,7 @@ public class Progress : Control
         /// <since_tizen> 6 </since_tizen>
         public Progress() : base()
         {
+            Initialize();
```

Comment:
Don't we need this Initialize in two other ctors?

Reply:
You are right!

Reply:
done